docs/settings-migration.md

```diff
@@ -121,3 +121,23 @@ The `terraform.experimentalFeatures` setting block has been extracted out to ind
 </table>
 
 For an easy to view table of settings, see the `Contributions` tab when viewing the Terraform Extension in the VS Code Extension Pane.
+
+## Setting Scope
```

Comment:
Nitpick: Just for extra clarity - could we also mention that this basically prevents the options below to be loaded from workspace/directory-level settings?

Reply:
Good idea. Amended